cellengine/resources/fcs_file.py

```diff
@@ -225,10 +230,16 @@ def events(self, events):
     def spill_string(self):
```

Comment:
This can be simplified to only have only one `return` and save a line or two.

Reply:
:+1: